lib/tasks/subscriptions/test.rake

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+namespace :ofn do
+  namespace :subs do
+    namespace :test do
+      desc "Repeat placement job for a specific Order Cycle"
+      task repeat_placement_job: :environment do
+        puts "WARNING: this task will generate new, and potentially duplicate, customer orders"
+
+        order_cycle_id = request_order_cycle_id
+
+        # Open Order Cycle by moving close_at to the future and open_at to the past
+        set_order_cycle_times(order_cycle_id,
+                              Time.zone.now - 15.minutes,
+                              Time.zone.now + 15.minutes)
+
+        # Reset Proxy Orders of the Order Cycle
+        #   by detatching them from existing orders and resetting placed and confirmed dates
+        ProxyOrder.find_by(order_cycle_id: order_cycle_id).update(order_id: nil,
```

Comment:
It would be good to get feedback if this failed.
```suggestions
        ProxyOrder.find_by(order_cycle_id: order_cycle_id).update!(order_id: nil,
```